Completed Feature Set

rsundar opened the pull request:
## FEATURES ADDED

- Created an app using Create React App,
- Got covid19 data from the Narrativa API,
- Built a Hompage component comprising  a list of country names and total case count,
- Built a Country specific detail page comprising region wise breakdown of cases,
- Created unique routes for each country.

oloomoses reviewed (changes requested):
Hi @rsundar, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [ ] Kindly add mock tests for the API and the functions you have created as per the project requirements.

_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

_Slack: Oloo Moses_
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

rsundar commented on src/Components/home.jsx:
```diff
@@ -0,0 +1,92 @@
+      <input type="text" onChange={filterValues} className="form-control no-shadow" />
+    </InputGroup>
+  );
+};
```
Moved to FilterInput.jsx and imported as needed

rsundar commented on src/App.js:
```diff
@@ -1,24 +1,38 @@
+          </Link>
+          <h3 className="m-0">COVID-19 CASELOAD</h3>
+        </Navbar>
       </header>
```
Done

rsundar commented on src/redux/configStore.js:
```diff
@@ -0,0 +1,11 @@
+const store = createStore(
+  covidDataReducer,
+  applyMiddleware(thunkMiddleware, logger),
+);
```
Done

rsundar commented:
## CHANGES MADE

- Moved the Api calls into a saperate utils folder for better maintainability,
- Moved the filter into a saperate component and imported as needed,
- Add the required functional icons on the nav bar,
- Added tests for the Api calls
